tests/onnxruntime/test_modeling.py

```diff
@@ -938,9 +938,12 @@ def test_stable_diffusion_model_on_rocm_ep_str(self):
         self.assertListEqual(model.providers, ["ROCMExecutionProvider", "CPUExecutionProvider"])
 
     def test_load_model_from_hub_private(self):
-        subprocess.run("huggingface-cli logout", shell=True)
-        # Read token of fxmartyclone (dummy user).
-        token = "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"
+        token = os.environ.get("FXMARTYCLONE_READ_TOKEN", None)
+
+        if token is None:
+            self.skipTest(
+                "Test requires a token for fxmartyclone in the environment variable `FXMARTYCLONE_READ_TOKEN`."
+            )
 
         model = ORTModelForCustomTasks.from_pretrained("fxmartyclone/tiny-onnx-private-2", use_auth_token=token)
```

Comment:
And perhaps move the repo in an org w/ multiple people, do you have an optimum org on the hub?

Reply:
yes would be nice to move everything to https://huggingface.co/organizations/optimum-internal-testing, just send you an invite @fxmarty

Reply:
added one https://huggingface.co/optimum-internal-testing/tiny-random-phi-private (onnx revision)